packages/web-components/src/utils/date-utils.ts

```diff
@@ -177,7 +177,8 @@ export function validate(
     component.invalidDay = false;
   }
 
-  if (month && (month < minMonths || month > maxMonths)) {
+  if ((month && (month < minMonths || month > maxMonths)) ||
```

Comment:
Is it possible to add some clarifying comments like the other if statements have in this function?

Reply:
I added some comments which should help - let me know what you think.